.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# Global code owners of the repository:
+# (Approval from a user/group is required to merge into main)
+*       @harmony-das-maintainers
```

Comment:
Is the `+` meant to be here? I've seen just a raw `*` without that plus sign in others,